From 5a24286ed6315e1fef755ca1970792554f59b1fc Mon Sep 17 00:00:00 2001
From: David Bauer <mail@david-bauer.net>
Date: Wed, 26 May 2021 22:15:35 +0200
Subject: [PATCH] ACS: fix channel 100 frequency

Channel 100 is a valid channel to choose for 80MHz operation. However,
it's assigned to 5500 MHz, not 5550MHz. In fact, there is no channel
assigned to this frequency.

Fix this obbvious typo to allow ACS to select channel 100 for 80 MHz
operation again.

Fixes commit bef5eee4f7b2 ("Convert channel to frequency based selection for AP mode ACS")

Signed-off-by: David Bauer <mail@david-bauer.net>
---
 src/ap/acs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/src/ap/acs.c
+++ b/src/ap/acs.c
@@ -386,7 +386,7 @@ static int acs_usable_bw40_chan(const st
 
 static int acs_usable_bw80_chan(const struct hostapd_channel_data *chan)
 {
-	const int allowed[] = { 5180, 5260, 5550, 5580, 5660, 5745, 5955, 6035,
+	const int allowed[] = { 5180, 5260, 5500, 5580, 5660, 5745, 5955, 6035,
 				6115, 6195, 6275, 6355, 6435, 6515, 6595, 6675,
 				6755, 6835, 6915, 6995 };
 	unsigned int i;
